assignment.html

```diff
@@ -0,0 +1,84 @@
+<html>
+    <head>
+        <style>
+            figure
+            {
+                float:left;
+                border-left: 1px solid rgba(228, 223, 223, 0.87);
+                height: 300px;
+                margin-left:-50px;
+                margin-right:59px;
+                margin-top:-20px; 
```

Comment:
using negative margin is not a good idea. So try to implement without negative margin.